src/main/java/org/jenkinsci/plugins/gitclient/ChangelogCommand.java

```diff
@@ -121,4 +121,10 @@
      * ChangelogCommand instance or files will be left open.
      */
     void abort();
+
+    /**
+     * Include merge commits in the changelog
+     * @param flag
```

Comment:
Please provide a few words of description of the purpose of flag so that the Javadoc processor with Java 8 will not add new javadoc warnings